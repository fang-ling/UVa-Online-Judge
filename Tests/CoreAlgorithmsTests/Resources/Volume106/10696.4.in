1
91
101
0
