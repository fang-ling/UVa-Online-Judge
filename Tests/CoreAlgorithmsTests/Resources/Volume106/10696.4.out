f91(1) = 91
f91(91) = 91
f91(101) = 91
